.github/workflows/dapr-dev-container.yml

```diff
@@ -15,11 +15,14 @@ name: dapr-dev-container
 
 on:
   workflow_dispatch:
+  schedule:
+    # Run weekly on Tuesdays
+    - cron: "22 03 * * 2"
```

Comment:
What's with 3:22 though? 😄

Reply:
A random time that is not on top of the hour (when a lot of concurrent tasks are happening), and that’s in the middle of the night PST, when we see less activity on the repo.
I actually put a bit of thought on this :)

Reply:
Genesis 3:22

And the Lord God said, “The man has now become like one of us, knowing good and evil. He must not be allowed to reach out his hand and take also from the tree of life and eat, and live forever.”